### ISSUE TITLE
Everything stops working after the new update wend piloting ships

### ISSUE THREAD

**Issue by Reason147** [tsunamayo/Starship-EVO#3688]
### Short Description of the issue:

...Everything stops working after the new update wend piloting ships, all my tips worked fine but on larger ships and detailed ships the game just stops everything frizes up I can't move buttons don't work can't even exit the ship without forcing a task manager shutdown of EVO.  IDK why this keeps happening but I can't build like this.  My PC is more than powerful to run EVO so I shouldn't be seeing this at all. I'm running a Decktop ADM A10-7800 Radeon R7 12 Core 4C+BG 3.50GHz 12GB RAM & NVIDIA GeForce GTX 1050 

### Steps to reproduce the issue:

...I wish I knew 

### Environment
- Starship EVO version: New build - EXPERIMENTAL] 21w12b: Hotfixes
- Graphics Card: NVIDIA GeForce GTX 1050

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `Player.log` file. You can find the log file here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

If your issue concerns a particular design please include any blueprint that could help to reproduce the issue.
Blueprint folder is over at:
`%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Blueprints\`
You will need to zip the folder to upload it. 
To do so Right click and choose [Send to -> compressed zip file] then drag the .zip onto your report to upload it.

**Comment by ProPeach** [comment]
Can you send your Player.log file? It would be really helpful in finding out what the issue is. There's instructions on how to find it in the Additional Information part of your post

**Comment by Reason147** [comment]
[Player.zip](https://github.com/tsunamayo/Starship-EVO/files/6217612/Player.zip)

**Comment by Reason147** [comment]
I sent you all the text files I had I< I'm going try a Delete and restore of my charter profiles form aback up the see if the helps I have a backup or everything before the bug hit. I'm going to see if it's a player charter problem or not.

**Comment by Reason147** [comment]
Strange Glitch, War core toggle off/on freezes game
[https://www.youtube.com/watch?v=nAju6Ri6mW0](url)
[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/6217657/output_log.txt)
[Player.log](https://github.com/tsunamayo/Starship-EVO/files/6217658/Player.log)
[Player-prev.log](https://github.com/tsunamayo/Starship-EVO/files/6217659/Player-prev.log)

**Comment by tsunamayo** [comment]
@Reason147 Hi, sorry for the super late answer this case was lost during the big update. Do you still have issues? It seems related to some turret on your blueprint. Could you send me your ship? Thanks

**Comment by Reason147** [comment]
Yes it's still a problem, I have more than one ship that is doing this, the works ones are as listed below. 2 are on the steam workshop and 2 are private builds. 

SR68 Bounty Class (Outlander): https://steamcommunity.com/sharedfiles/filedetails/?id=2445691048 (upgrades pending)

The worst ship crashes the game making it unusable and had been rebuilt with no change to the problem, It seems some have this bug while others do not. 
[EVOShips-Reason.zip](https://github.com/tsunamayo/Starship-EVO/files/6817891/EVOShips-Reason.zip)
Here the other ship with the known bug, the "Training Ship" is the worst one it crashes the game even after 3 rebuilds.

the Doom Empire Patrol ship and Harland also have the bug but after 4 or 5 restarts of EVO will work.
